SwiftUIDemoAppUITests/Tests/DeeplinkTests.swift

```diff
@@ -38,7 +38,7 @@ final class DeeplinkTests: StreamTestCase {
                 .replacingOccurrences(of: " ", with: "")
 
             let prodAndStaging = """
-            {"applinks":{"apps":[],"details":[{"appID":"EHV7XZLAHA.io.getstream.iOS.VideoDemoApp","paths":["/video/demos/*","/video/demos"]}]}}
+            {"applinks":{"apps":[],"details":[{"appIDs":["EHV7XZLAHA.io.getstream.iOS.VideoDemoApp","EHV7XZLAHA.io.getstream.rnvideosample"],"paths":["/video/demos/*","/video/demos"]}]}}
```

Comment:
why do we add rn sample?

Reply:
Because on BE the associated file has been updated to support RN push notifications.